README.md

```diff
@@ -6,8 +6,10 @@
 ## .NET SDK
 
 <!-- x-hide-in-docs-end -->
+<!-- x-release-please-start-version -->
 [![Specification](https://img.shields.io/static/v1?label=specification&message=v0.5.2&color=yellow&style=for-the-badge)](https://github.com/open-feature/spec/releases/tag/v0.5.2)
-[![Release](https://img.shields.io/static/v1?label=release&message=v1.4.0&color=blue&style=for-the-badge)](https://github.com/open-feature/dotnet-sdk/releases/tag/v1.4.0) <!-- x-release-please-version --> <!-- x-release-please-version -->
+[![Release](https://img.shields.io/static/v1?label=release&message=v1.4.0&color=blue&style=for-the-badge)](https://github.com/open-feature/dotnet-sdk/releases/tag/v1.4.0)
```

Comment:
```suggestion
<!-- x-release-please-start-version -->
[![Release](https://img.shields.io/static/v1?label=release&message=v1.4.0&color=blue&style=for-the-badge)](https://github.com/open-feature/dotnet-sdk/releases/tag/v1.4.0)
```

Reply:
I tried this but this causes the badges to be displayed vertically instead of horizontally. Let me play around with it for a few minutes to see if I can find a workaround.